gcsio/src/test/java/com/google/cloud/hadoop/gcsio/CreateFileOptionsTest.java

```diff
@@ -25,20 +26,44 @@
 @RunWith(JUnit4.class)
 public class CreateFileOptionsTest {
   @Test
-  public void testChecksForContentTypeAttributes() {
+  public void validOptions() {
     CreateFileOptions.builder()
-        .setOverwriteExisting(true)
-        .setContentType("")
-        .setAttributes(ImmutableMap.of("Innocuous", "".getBytes(UTF_8)))
+        .setAttributes(ImmutableMap.of("Innocuous", "text".getBytes(UTF_8)))
+        .setContentType("text")
+        .setWriteMode(CreateFileOptions.WriteMode.OVERWRITE)
+        .setOverwriteGenerationId(0)
         .build();
+  }
+
+  @Test
+  public void invalidOptions_contentType_setViaAttributes() {
```

Comment:
we seem to be using camelCase for test names?

Reply:
New test names follow test naming convention that use underscores to separate logical parts of the test name: https://medium.com/@stefanovskyi/unit-test-naming-conventions-dd9208eadbea